src/scenes/CreateEvent/index.js

```diff
@@ -0,0 +1,652 @@
+                {intl.formatMessage(messages.title)}
+              </Grid>
+              <Grid item container xs={9}>
+                <Grid item xs={12}>
+                  <Field
+                    name="name"
+                    placeholder={intl.formatMessage(messages.namePlaceholder)}
+                    validate={[this.validateTitleLength]}
+                    component={this.renderTextField}
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.betStartTime)}
+              </Grid>
+              <Grid item container xs={9}>
+                <Field
+                  name={ID_BETTING_START_TIME}
+                  validate={[this.validateTimeAfterNow]}
+                  component={this.renderDateTimePicker}
+                />
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.betEndTime)}
+              </Grid>
+              <Grid item container xs={9}>
+                <Field
+                  validate={[this.validateTimeAfterNow]}
+                  name={ID_BETTING_END_TIME}
+                  component={this.renderDateTimePicker}
+                />
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.resultSetStartTime)}
+              </Grid>
+              <Grid item container xs={9}>
+                <Field
+                  validate={[this.validateTimeAfterNow]}
+                  name={ID_RESULT_SETTING_START_TIME}
+                  component={this.renderDateTimePicker}
+                />
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.resultSetEndTime)}
+              </Grid>
+              <Grid item container xs={9}>
+                <Field
+                  validate={[this.validateTimeAfterNow]}
+                  name={ID_RESULT_SETTING_END_TIME}
+                  component={this.renderDateTimePicker}
+                />
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.outcomes)}
+              </Grid>
+              <Grid item xs={9}>
+                <FieldArray name="outcomes" component={this.renderOutcomeList} />
+              </Grid>
+            </Grid>
+            <Grid container>
+              <Grid item xs={3}>
+                {intl.formatMessage(messages.resultSetter)}
+              </Grid>
+              <Grid item xs={9}>
+                <Field
+                  required
+                  fullWidth
+                  name="resultSetter"
+                  placeholder="Enter the address of result setter or select your own address"
```

Comment:
Use localized string. And, grammar change: `Enter the address of the result setter or select your own address`